### DISCUSSION TITLE
Can i simulate F10?

### DISCUSSION THREAD

**Discussion by FahriDevZ** [Schneegans/Fly-Pie#131] [Fly-Pie Q&A] [answered]
I tried f1 - f12, but f10 can't be simulated and instead focuses on the exclamation mark.
Anyone here have a solution?

version: fly-pie 8

Thanks. :)

**Comment by Schneegans** [comment, marked as answer]
That's interesting. I can indeed reproduce the behavior. And I can observe a similar behavior in other GTK applications: <kbd>F10</kbd> seems to move the input focus to something in the headerbar. I'll check whether this can be disabled somehow...

In the meantime, you can directly edit the menu configuration:
* Choose another key for your action, e.g. <kbd>F9</kbd>
* Close the menu editor.
* Open the `dconf-editor` and navigate to `org/gnome/shell/extensions/flypie/menu-configuration`.
* Search and replace <kbd>F9</kbd> by  <kbd>F10</kbd> in the configuration. You may want to copy the entire menu configuration to a text editor, e.g. `gedit` to make this easier.
* Hit the "Apply button".

**Comment by Schneegans** [comment]
I just chatted with the GTK devs and the F10 shortcut is somewhat hardwired in GTK4. So there's not much we can do about it. If you really need to use F10, you will have to edit the menu configuration by hand as I described above!

**Reply by FahriDevZ** [reply]
My suggestion might be to create a text input feature to solve some problems like this, for now I'm using your answer. Thank you.
